protocol/src/main/java/io/camunda/zeebe/protocol/record/value/JobRecordValue.java

```diff
@@ -38,6 +38,12 @@
   /** @return remaining retries */
   int getRetries();
 
+  /**
+   * @return the time of backoff in milliseconds. If backoff is disabled this method returns 0
+   *     (default value).
+   */
+  long getRetryBackOff();
```

Comment:
:x: By adding a new property to the record, we need to adjust the following places:

* Change the JSON format of the record in `io.camunda.zeebe.protocol.impl.JsonSerializableToJsonTest`. This test verifies that we don't break the JSON format of the records accidentally.
* Update the ElasticSearch exporter schema. The exporter would fail because the schema is defined strictly here:
  * `exporters/elasticsearch-exporter/src/main/resources/zeebe-record-job-template.json`
  * `exporters/elasticsearch-exporter/src/main/resources/zeebe-record-job-batch-template.json`